testing/test_full.py

```diff
@@ -3145,3 +3145,73 @@ def test_generic_data_split():
     sanitize_outputDir()
 
     print("passed")
+
+
+def test_dynunet_model(device):
+    # is this test correct ??
+
+    # for more info : https://github.com/Project-MONAI/MONAI/blob/96bfda00c6bd290297f5e3514ea227c6be4d08b4/tests/test_dynunet.py
+    print("51: Starting test for  Dynunet implementation")
+
+    # Reading the parameters
+    parameters = parseConfig(
+        testingDir + "/config_segmentation.yaml", version_check_flag=False
+    )
+    training_data, parameters["headers"] = parseTrainingCSV(
+        inputDir + "/train_3d_rad_segmentation.csv"
+    )
+    parameters["model"]["architecture"] = "dynunet"
+    parameters["model"]["dimension"] = 3
+    parameters["patch_size"] = patch_size["3D"]
+    parameters["model"]["kernel_size"] = [3, 3, 3, 1]
+    parameters["model"]["strides"] = [1, 1, 1, 1]
+    sanitize_outputDir()
+    TrainingManager(
+        dataframe=training_data,
+        outputDir=outputDir,
+        parameters=parameters,
+        device=device,
+        resume=False,
+        reset=True,
+    )
+
+    sanitize_outputDir()
+
+    print("passed")
+
+
+def test_dynunet_mandatory_input_values():
+    """
+    It tests the mandatatory inputs for dynunet model exist in the parameters["model"].
+    Mandotary inputs:
+     - kernel_size
+     - strides
+
+     It checks If an AssertionError is raised.
+    """
+    # Reading the parameters from segmentation config.
+
+    with pytest.raises(AssertionError):
+        # Reading the parameters from segmentation config.
+        parameters = parseConfig(
+            testingDir + "/config_segmentation.yaml", version_check_flag=False
+        )
+
+        parameters["model"]["architecture"] = "dynunet"
+
+        training_data, parameters["headers"] = parseTrainingCSV(
+            inputDir + "/train_3d_rad_segmentation.csv"
+        )
+
+        sanitize_outputDir()
+        TrainingManager(
+            dataframe=training_data,
+            outputDir=outputDir,
+            parameters=parameters,
+            device=device,
+            resume=False,
+            reset=True,
+        )
+
+        sanitize_outputDir()
+        print("Passed")
```

Comment:
Instead of having separate test for `dynunet`, can you append to the list initialized using `all_models_segmentation` [[ref](https://github.com/mlcommons/GaNDLF/blob/new-apis_v0.1.0-dev/testing/test_full.py#L51)]? This will test for all segmentation cases (2D, 3D radiology and histology).